<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>366</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>278</width>
    <height>366</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>278</width>
    <height>366</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>278</width>
    <height>366</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rozpoznawanie Alkoholi</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="StackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-10</y>
        <width>231</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string>Witaj!</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Pytanie1">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1. Proszę podać kolor napoju</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>bezbarwny</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>biały</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>żółty</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>złoty</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>brązowy</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>czerwony</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>czarny</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>zielony</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>niebieski</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Pytanie2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>2. Proszę podać stężenie alkoholu:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Pytanie3">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>3. Proszę podać jaki posiada zapach:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox31">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>eteryczny</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox32">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>słodki</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox33">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>korzenny</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox34">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>siarkowy</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox35">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ziołowy</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox36">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>spirytusowy</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Pytanie4">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>4. Czy napój jest gazowany?</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox41">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Tak</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Pytanie5">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>5. Proszę podać kontynent pochodzenia:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox51">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Europa</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox52">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Azja</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox53">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Ameryka Północna</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox54">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Ameryka Południowa</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox55">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Afrya</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox56">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Australia</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Pytanie6">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>6. Prosze podać smak:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox63">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>gorzki</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox64">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>słodki</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox65">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>kwaśny</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox61">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>słony</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox62">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>bezsmakowy</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dalej</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wstecz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Resetuj</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
